authors/apps/articles/serializers.py

```diff
@@ -1,8 +1,15 @@
 from rest_framework import serializers
-from .models import Article
+from authors.apps.article_tag.models import ArticleTag
+from .models import Article, FavoriteArticle
+from authors.apps.authentication.serializers import UserSerializer
 
 class ArticleSerializer(serializers.ModelSerializer):
     """This serializer requests and creates a new article"""
+    article_tags = serializers.SlugRelatedField(
```

Comment:
well done @llwasampijja, I have noticed that the key for tags in your request body and response does not match what is specified in the README. Is it okay if you update it to `tagList` to match the one specified

Reply:
ok